mat/format.go

```diff
@@ -173,25 +173,25 @@ func format(m formattable, prefix string, margin int, dot byte, squeeze bool, fs
 		fmt.Fprintf(fs, "Dims(%d, %d)\n", rows, cols)
 	}
 
+	var buf = make([]byte, 0, 4)
```

Comment:
`buf := make([]byte, 0, 4) // Four bytes required because of unicode decoration.`